llvm/test/CodeGen/SPARC/inlineasm-bad.ll

```diff
@@ -13,8 +14,20 @@ entry:
 }
 
 ; CHECK-label:test_twinword_error
+; CHECK: error: invalid operand in inline asm: 'rd %asr5, ${0:L}
```

Comment:
Why are there now three copies of all this?

Reply:
https://github.com/llvm/llvm-project/commit/697dd93ae30f489e5bcdac74c2ef2d876e3ca064#r140730056
I reached out to author, he intended it.